### PR TITLE
Split talairach registration into recon_surf/talairach-reg.sh and python not found error

### PR DESCRIPTION
**Pull request by dkuegler**
run_fastsurfer.sh
- check for existence of python with error message
- move recon-surf flags into surf_flags array recon_surf/recon-surf.sh
- remove talariach registration from script 
Docker/build.py
- fix build error without attestation

### CODE REVIEW

**Comment by dkuegler** [comment]
This version is tested for three settings, `--seg_only`, `--seg_only --tal_reg` as well as `--tal_reg`. Seems to be working for all :).

**Comment by dkuegler** [comment]
This PR also provides better error managing if the python version does not match as reported in #464 .

File: run_fastsurfer.sh [hunk 1]
```diff
@@ -60,14 +60,7 @@ run_asegdkt_module="1"
 run_cereb_module="1"
 threads="1"
 # python3.10 -s excludes user-directory package inclusion, but passing "python3.10 -s" is not possible
```
Comment (m-reuter) [inline run_fastsurfer.sh]: here it still says that python.. -s is not possible, but below you define it like that?

Comment (dkuegler) [reply run_fastsurfer.sh]: Yeah, I fixed that in this commit

Comment (m-reuter) [reply run_fastsurfer.sh]: No, you did not fix this comment , Line 62, which is still in here.,

Comment (dkuegler) [reply run_fastsurfer.sh]: What I meant was
This comment is outdated. It used to be true, but this PR fixes the underlying issue

Comment (dkuegler) [reply run_fastsurfer.sh]:
```suggestion
# python3.10 -s excludes user-directory package inclusion
```

**Pull request merged**